src/h_vialib/client.py

```diff
@@ -31,15 +38,16 @@ def is_pdf(self):
 class ViaClient:  # pylint: disable=too-few-public-methods
     """A small wrapper to make calling Via easier."""
 
-    def __init__(self, service_url, host_url, secret):
+    def __init__(self, host_url, secret, service_url=None, html_service_url=None):
```

Comment:
This is the breaking change. If the caller used positional args it's bust.

This change allows us to:

 * Deploy to people who used `kwargs` without breaking anything
 * Not force the user to fill out nonsense values they aren't interested in (for example LMS doesn't yet want to talk to ViaHTML directly... so why force it to use the value)

Reply:
silly nitpick: If we are going to break it we might as well have the secret as the first arg and all all the urls together, it just seems neater

Reply:
Yep. I'll swap it around